opensanctions/crawlers/md_rise_profiles.py

```diff
@@ -0,0 +1,107 @@
+from urllib.parse import urljoin, urlencode
+from normality import stringify, collapse_spaces, slugify
+
+from opensanctions.core import Context
+from opensanctions import helpers as h
+import re
+
+COUNTRY = "md"
+PERSON_TYPES = {
+    "bailiff",
+    "businessman",
+    "businesswoman",
+    "cna-director",
+    "cna-officer",
+    "csm-member",
+    "csm-member",
+    "director",
+    "diplomat",
+    "former-governor",
+    "former-prosecutor",
+    "investigator",
+    "inj-director",
+    "judge",
+    "lawyer",
+    "minister",
+    "police-officer",
+    "politician",
+    "procuror",
+    "prosecutor",
+    "public-officer",
+    "russian-politician",
+    "sis-officer",
+}
+
+
+def parse_date(text):
+    return h.parse_date(text, ["%d.%m.%Y"])
+
+
+def crawl_entity(context: Context, relative_url: str):
+    url = urljoin(context.source.data.url, relative_url)
+    doc = context.fetch_html(url)
```

Comment:
We could add a little `cache_days=14` here without risk of missing major world events.